### ISSUE TITLE
Swagger UI is blank

### ISSUE THREAD

**Issue by averri** [micronaut-projects/micronaut-openapi#919]
### Expected Behavior

The Swagger UI should be displayed when accessing its URL via browser (ending in `/swagger-ui`).

### Actual Behaviour

The Swagger UI is blank:

![image](https://user-images.githubusercontent.com/1898447/215295590-b64dd04c-78ab-4eaf-8698-d3dcb4fb3a1c.png)

The Chrome debug console shows the following errors:

![image](https://user-images.githubusercontent.com/1898447/215295622-8e9c80d4-8ef4-44fe-9e1f-c60b0a990012.png)

### Steps To Reproduce

I have followed the guidelines described here: https://micronaut-projects.github.io/micronaut-openapi/latest/guide/

Compiled the project using `./gradlew build` to produce the OpenAPI artifacts and then started the application with `./gradlew run`.

### Environment Information

- Operating System: Windows 11
- Java Runtime:
```
openjdk 17.0.5 2022-10-18
OpenJDK Runtime Environment GraalVM CE 22.3.0 (build 17.0.5+8-jvmci-22.3-b08)
OpenJDK 64-Bit Server VM GraalVM CE 22.3.0 (build 17.0.5+8-jvmci-22.3-b08, mixed mode, sharing)
```

- `gradle.properties`:
```
micronautVersion=3.8.1
kotlinVersion=1.8.0
```

- `src/main/resources/application.yml`:
```yaml
micronaut:
  server:
    port: 5000
  data:
    pageable:
      max-page-size: 50
  metrics:
    enabled: true
    export:
      prometheus:
        enabled: true
        step: PT15S
        descriptions: true
  router:
    static-resources:
      swagger:
        paths: classpath:META-INF/swagger
        mapping: /swagger/**
      swagger-ui:
        paths: classpath:META-INF/swagger/views/swagger-ui
        mapping: /swagger-ui/**
```

- `build.gradle.kts`:
```kotlin
plugins {
    id("org.jetbrains.kotlin.jvm") version "1.8.0"
    id("org.jetbrains.kotlin.kapt") version "1.8.0"
    id("org.jetbrains.kotlin.plugin.allopen") version "1.8.0"
    id("groovy")
    id("com.github.johnrengelman.shadow") version "7.1.2"
    id("io.micronaut.application") version "3.7.0"
    id("io.micronaut.aot") version "3.7.0"
}

version = "0.1"
group = "com.strenux"

val kotlinVersion = project.properties["kotlinVersion"]

repositories {
    mavenCentral()
}

dependencies {
    compileOnly("org.graalvm.nativeimage:svm")

    //    implementation("io.micronaut.security:micronaut-security-annotations")
    implementation("io.micronaut.data:micronaut-data-jdbc")
    implementation("io.micronaut.email:micronaut-email-javamail")
    implementation("io.micronaut.email:micronaut-email-template")
    implementation("io.micronaut.flyway:micronaut-flyway")
    implementation("io.micronaut.kotlin:micronaut-kotlin-runtime")
    implementation("io.micronaut.micrometer:micronaut-micrometer-registry-prometheus")
    implementation("io.micronaut.problem:micronaut-problem-json")
    implementation("io.micronaut.reactor:micronaut-reactor")
    implementation("io.micronaut.reactor:micronaut-reactor-http-client")
    implementation("io.micronaut.security:micronaut-security-jwt")
    implementation("io.micronaut.security:micronaut-security-oauth2")
    implementation("io.micronaut.serde:micronaut-serde-jackson")
    implementation("io.micronaut.sql:micronaut-jdbc-hikari")
    implementation("io.micronaut.tracing:micronaut-tracing-jaeger")
    implementation("io.micronaut.views:micronaut-views-thymeleaf")
    implementation("io.micronaut:micronaut-http-client")
    implementation("io.micronaut:micronaut-management")
    implementation("io.micronaut:micronaut-validation")
    implementation("io.temporal:temporal-sdk:1.17.0")
    implementation("jakarta.annotation:jakarta.annotation-api")
    implementation("jakarta.persistence:jakarta.persistence-api:2.2.3")
    implementation("org.apache.commons:commons-lang3:3.12.0")
    implementation("org.http4k:http4k-client-apache:4.37.0.0")
    implementation("org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}")
    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlinVersion}")
    implementation("org.slf4j:slf4j-api:2.0.6")
    implementation("org.springframework.security:spring-security-crypto:6.0.1")
    implementation("io.swagger.core.v3:swagger-annotations")


    kapt("io.micronaut.data:micronaut-data-processor")
    kapt("io.micronaut.serde:micronaut-serde-processor")
    kapt("io.micronaut:micronaut-http-validation")
    kapt("io.micronaut.openapi:micronaut-openapi:4.8.2")

    runtimeOnly("ch.qos.logback:logback-classic:1.4.5")
    runtimeOnly("com.fasterxml.jackson.module:jackson-module-kotlin")
    runtimeOnly("commons-logging:commons-logging:1.2")
    runtimeOnly("org.postgresql:postgresql")
}

application {
    mainClass.set("com.strenux.app.Application")
}

java {
    sourceCompatibility = JavaVersion.VERSION_17
    targetCompatibility = JavaVersion.VERSION_17
}

tasks {
    compileKotlin {
        kotlinOptions {
            jvmTarget = "17"
        }
    }
    compileTestKotlin {
        kotlinOptions {
            jvmTarget = "17"
        }
    }
}

graalvmNative.toolchainDetection.set(false)

micronaut {
    runtime("netty")
    testRuntime("spock2")
    processing {
        incremental(true)
        annotations("com.strenux.*")
    }
    aot {
        // Please review carefully the optimizations enabled below
        // Check https://micronaut-projects.github.io/micronaut-aot/latest/guide/ for more details
        optimizeServiceLoading.set(true)
        convertYamlToJava.set(true)
        precomputeOperations.set(true)
        cacheEnvironment.set(true)
        optimizeClassLoading.set(true)
        deduceEnvironment.set(true)
        optimizeNetty.set(true)
    }
    kapt {
        arguments {
            arg(
                "micronaut.openapi.views.spec",
                "-Dmicronaut.openapi.views.spec=rapidoc.enabled=true,swagger-ui.enabled=true"
            )
        }
    }
}

allOpen {
    annotations("io.micronaut.http.annotation.Controller")
}
```

### Version

3.8.1

**Comment by ericblue** [comment]
Hi @averri I was just checking out Micronaut for the 1st time today, and wrestled with a similar issue this evening, but on version 3.8.3.  This issue is happening even using the Micronaut Launch demo app (https://micronaut.io/launch/ - selecting 3.8.3 and adding OpenAPI and Swagger as features).

I also see the same console output on Chrome.

swagger-ui-bundle.js:2 Uncaught ReferenceError: module is not defined
    at swagger-ui-bundle.js:2:1060083
    at swagger-ui-bundle.js:2:1060095
(anonymous) @ swagger-ui-bundle.js:2
(anonymous) @ swagger-ui-bundle.js:2
swagger-ui:45 Uncaught ReferenceError: SwaggerUIBundle is not defined
    at window.onload (swagger-ui:45:22)

I'm not sure of the root cause but the generated version of swagger-ui-bundle.js has issues with the module export.

I verified that this problem does not exist in Micronaut version 3.9.0-SNAPSHOT.  And, in fact, copied the swagger-ui-bundle.js from that project over to 3.8.3 and the problem was resolved.  If you do a diff of the 2 files you can see the top of the file has some differences in the module declaration.

So until the root issue is resolved, try updating your gradle.properties:

micronautVersion=3.9.0-SNAPSHOT

Hopefully this helps.

- Eric

**Comment by altro3** [comment]
@averri Hi! Sorry, my fault. I accidentally replaced `swagger-ui-bundle.js` with `swagger-ui-es-bundle.js` in one of the latest updates.

Fixed here #920

**Comment by averri** [comment]
Hi guys, @altro3, and @ericblue, thank you so much for informing that. 
I'll wait for the next Micronaut release.
